src/utils/Utils.js

```diff
@@ -13,6 +13,31 @@ export default class Utils {
     return newArray;
   }
 
+  static loadUnusedPairs(allProps, properties) {
+    if (allProps === undefined || properties === undefined) return {};
+    const propMap = {};
+    properties.forEach((prop) => {
+      if (!propMap[prop.namespace] || propMap[prop.namespace] === undefined) propMap[prop.namespace] = [prop.name];
```

Comment:
please don't use 1 line `if` statements

```
if (...) {
  ...
}
else {
  ...
}
```

I'll have to check the ESLint in gallery.